app/models/account_automatic_topup_config.rb

```diff
@@ -1,4 +1,4 @@
 class AccountAutomaticTopupConfig < ApplicationRecord
   belongs_to :from_account, class_name: 'Account', foreign_key: :from_account_id
-  belongs_to :to_account, class_name: 'Account', foreign_key: :to_account_id
+  belongs_to :to_account, class_name: 'Account', foreign_key: :to_account_id, optional: true
```

Comment:
why it's optional?

Reply:
It was attempt to resolve the validation issue.